examples/.test/ssg/test/smoke.test.ts

```diff
@@ -0,0 +1,13 @@
+import { test } from '@playwright/test';
+
+test('query should be prefetched', async ({ page }) => {
+  await page.goto('/');
+
+  await page.waitForSelector('text=hello client');
+});
+
+test('dates should be serialized', async ({ page }) => {
```

Comment:
kinda shitty test cause it doesn't really test the serialization but if it passes it means we didn't get an error when accessing the date-method 🤷🏼‍♂️